app/code/Magento/CatalogGraphQl/Model/Resolver/Products/DataProvider/Customer/ContextProcessor.php

```diff
@@ -0,0 +1,64 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\CatalogGraphQl\Model\Resolver\Products\DataProvider\Customer;
+
+use Magento\Customer\Api\Data\GroupInterface;
```

Comment:
We should avoid hard dependency on Customer from CatalogGraphQl

Reply:
@pmarjan Please resolve this issue when you have a chance.